src/i18ndude/extract.py

```diff
@@ -436,6 +436,15 @@ def find_files(dir, pattern, exclude=()):
         folders = (dir, )
 
     def visit(files, dirpath, names):
+        # exclude dirnames
+        for ex in exclude:
+            # exact dirname match
+            if ex in dirpath.split("/"):
```

Comment:
Generally speaking `dirpath.split(os.path.sep)` will repeat equally in every cycle so a minor improvement would be moving this outside the for block